src/router.js

```diff
@@ -17,8 +17,14 @@ export default function (options) {
       }
     },
     hooks: {
+      onUpdate: function (oldModel, newModel, data, actions) {
+        if(newModel.router && oldModel.router.path !== newModel.router.path &&
```

Comment:
Yes but if it's the same object it will have the same path. We use a really simple merge/cloning algorithm and I'm not sure if this algorithm always guarantees that the compared paths will be different.